zetasql/parser/bison_parser.y

```diff
@@ -7648,6 +7766,7 @@ keyword_as_identifier:
     | "RETURNS"
     | "RETURN"
     | "REVOKE"
+    | "ROLE"
```

Comment:
pg merge `user` and `group` into single `role`, user just role with LOGIN attribute. This have many advantages, ref https://stackoverflow.com/a/8499491

```txt
allow groups to have groups as members

the ACL code would be simplified

the GRANT/REVOKE syntax and the display format for ACL lists could be simplified, since there'd be no need for a syntactic marker as to whether a given name is a user or a group.

In some circumstances I could see it making sense to allow logging in directly as a group/role/whatchacallit

This would also solve the problem that information_schema views will show only owned objects

[makes it easier to] representing privileges granted to groups [since you'd simply reuse the role related code?]
```

At least simplification make sense, e.g the privilege list. Consider ?